src/System.Management.Automation/FormatAndOutput/DefaultFormatters/Help_format_ps1xml.cs

```diff
@@ -413,7 +402,6 @@ internal static IEnumerable<ExtendedTypeDefinition> GetFormatData()
                 MamlPossibleValueControl,
                 MamlTrueFalseShortControl,
                 MamlIndentedSyntaxControl,
-                CommonWorkflowParametersControl,
```

Comment:
This entry is with the index 16 in the array `sharedControls`, so for an index that is greater than 16, we need to adjust it by subtracting 1. This is why you will see many index changes below.

Reply:
You could leave `null` in this entry.
In another PR - you could introduce names for each shared control to make the file more friendly to human editing now that we no longer use a tool to generate it.

Reply:
Will do in a separate PR #6712